src/actions/registerAction.js

```diff
@@ -0,0 +1,31 @@
+import { REGISTER_SUCCESS, REGISTER_FAILED, REGISTER_TRIGGER} from './types';
+import axiosConfig from '../config/axios';
+import { toastr } from 'react-redux-toastr'
+
+export const signUpUser = (userData) => (dispatch) => {
+	dispatch({ type: REGISTER_TRIGGER });
```

Comment:
Please make use of action creators when dispatching them.

Reply:
@unah254 do you use action creators when dispatching from the actions file? or from the components file?

Reply:
@bryan-munene, @unah254 is right. He needs to define an action creator then dispatch them.